multidim-interop/js/v0.41/test/ping.spec.ts

```diff
@@ -99,6 +99,13 @@ describe('ping test', () => {
           throw new Error("Failed to wait for dialer to finish")
         }
       }
+    } catch (err: any) {
+      if (err instanceof AggregateError) {
+        console.error(`unexpected exception in ping test: ${err}\n Errors: ${err.errors}`)
+      } else {
+        console.error(`unexpected exception in ping test: ${err}`)
+      }
```

Comment:
I am no JS expert but I think if you pass the `err` as an argument to `console.error`, it will display a nice stacktrace.
```suggestion
        console.error(`unexpected exception in ping test: ${err}\n Errors: ${err.errors}`)
      } else {
        console.error(`unexpected exception in ping test`, err)
      }
```

Reply:
Could be. Feel free to take over this PR by the way @MarcoPolo. You probably know better what you want to do with this suggestion and PR. My intention was more to share the need for better error logging in case the happy path is not being followed.